pelita/libpelita.py

```diff
@@ -274,7 +275,7 @@ def run_external_viewer(subscribe_sock, controller, geometry, delay):
     if delay:
         viewer_args += ["--delay", str(delay)]
 
-    tkviewer = os.path.join(os.path.dirname(sys.argv[0]), "tkviewer.py")
+    tkviewer = os.path.join(os.path.dirname(sys.argv[0]), "pelita-tkviewer")
```

Comment:
hmm, this is dangerous (in Debian for example `*.py` and `*.pyc` files get installed in different directory structures, and the `dirname` call may return the wrong dir)… can't this be addressed like in line 80 with a script in the `pelita.scripts` namespace?